1209/2023/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: T-Display rp2040
+owner: LilyGO
```

Comment:
Mind having yourself as the owner and link to the fork of CircuitPython with the changes? I want it to be clear that this isn't for LilyGo. It is for a third party using their hardware. Thanks!

Reply:
@kreier Awaiting your changes as requested by @tannewt. Then we can proceed with the board PR. Thanks.

Reply:
@tannewt sure, no problem. Yet I have a lot of options and possible answers, maybe you can point to the best one. Following the instructions on https://pid.codes/howto/ I have

### 2. Set up your organization page
a) I keep org/LilyGO/index.md
b) I create a new page for myself, linking to https://kreier.org

### 3. Find a VID and PID - 1209/2023/index.md
a) Keep the title in 1209/2023/index.md 
b) Change title to 'SSIS:bit with T-Display rp2040' in reference to my older project https://github.com/ssisbit/ssis.bit 
c) Change the owner to Matthias Kreier
d) Change the site to https://github.com/adafruit/circuitpython/pull/6037 
e) Keep the source with https://github.com/Xinyuan-LilyGO/LILYGO-T-display-RP2040
f) Change the source to https://github.com/kreier/t-display where I collect ideas since 2020

Reply:
> @tannewt sure, no problem. Yet I have a lot of options and possible answers, maybe you can point to the best one. Following the instructions on https://pid.codes/howto/ I have
> ### 2. Set up your organization page
> 
> a) I keep org/LilyGO/index.md b) I create a new page for myself as org/kreier/index.md , linking to https://kreier.org c) I create one for our school club org/SSISRobotics/index.md and link to https://github.com/ssis-robotics

b or c is fine. Whichever you'd like.

> ### 3. Find a VID and PID - 1209/2023/index.md
> 
> a) Keep the title in 1209/2023/index.md b) Change title to 'SSIS:bit with T-Display rp2040' in reference to my older project https://github.com/ssisbit/ssis.bit c) Change the owner to Matthias Kreier d) Change the site to [adafruit/circuitpython#6037](https://github.com/adafruit/circuitpython/pull/6037) e) Keep the source with https://github.com/Xinyuan-LilyGO/LILYGO-T-display-RP2040 f) Change the source to https://github.com/kreier/t-display where I collect ideas since 2020 g) Change the title to 'SSIS.bit with T-Display rp2040' and reference https://github.com/ssis-robotics/ssis.bit

The title can be the same but I'd switch the site to the CP PR. Please change the owner to match 2.

My main goal is to not give the impression that the manufacturer of the closed source board was granted a PID.

Reply:
@tannewt Thanks for the feedback, that's a good solution to make a proper use of the pidcodes project. I updated both the organization description (LilyGO is out and a new [SSIS-Robotics](https://github.com/kreier/pidcodes.github.com/blob/master/org/SSIS-Robotics/index.md) in) and the [1209/2023/index.md](https://github.com/kreier/pidcodes.github.com/blob/master/1209/2023/index.md). I left the title as _T-Display rp2040_ since the organization name will be prefixed in the final list.

Reply:
BTW: Thanks for volunteering to run this project. Please don't burn out. 🙂

Reply:
@kreier First of all thanks for get this through. Do you mind if I add a PID request for the T-Display PICOC3 board under your org "SSS Robotics" just like the T-Display RP2040 so I can try to contribute the board definition to CircuitPython as well. That board has wifi build-in should be able to power some very interesting project. Or if you have other suggestion for obtain a official PID?

Reply:
Hi @erongd , sure, no problem. You can copy the SSIS Robotics information for another PID request. Let's see if that is necessary. I think you should be able to use the same PID for the PicoC3 (depending how you plug in the USB-C cable) since in one side it's a regular T-Display rp2040 (only 2 serial pins connected to another onboard MCU) and the other way it is covered by Espressif - and not relevant for us, since we won't use it for CircuitPython on the ESP32 C3.